### ISSUE TITLE
Would it be at all possible to make it so that middle-click on the tab bar opens the last closed tab?

### ISSUE THREAD

**Issue by Lola-Lilypistol** [andy-portmen/undo-closed-tabs-button#13]
God, this would be lovely.

**Comment by jaegerschnitzel** [comment]
Yes or maybe it's possible to add an option which changes the default behaviour. Left or middle click restores last closed tab and right click shows drop-down list?

**Comment by derdershat** [comment]
oh yes please add this to this nice addon! 
had the restore last tab feature with session manager addon in Firefox <57 and it would be lovely to see this here
